.vscode/settings.json

```diff
@@ -16,6 +16,14 @@
       "source.organizeImports": "always"
     }
   },
+  "[javascript][typescript][typescriptreact][tailwindcss]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+      "source.fixAll": "always",
+      "source.organizeImports": "always"
+    },
+  },

```

Comment:
set as default formatter `prettier` for {.js, .ts, .tsx and .css files}

reference https://code.visualstudio.com/docs/getstarted/settings#_multiple-languagespecific-editor-settings